README.md

```diff
@@ -91,19 +92,19 @@ Please refer to [CHANGELOG](CHANGELOG.md)
 
 The following table lists the most recent few versions of the operator.
 
-| Operator Version | API Version | Kubernetes Version |
```

Comment:
Did you use some format tools on this kind of format issue?

Reply:
Yes, I am using [Prettier](https://github.com/prettier/prettier) to automatically format these docs.
Does it sounds good to you ?